config/default.json

```diff
@@ -17,12 +17,12 @@
   },
   "file_watcher_config": {
     "dir": [
-      "./src"
+      "tests/assets/test_folder"
```

Comment:
Please duplicate `default.json` content into `test.json` and use the second for the tests

Reply:
As discussed on discord, since we don't ship the code with the executable it's better to put the test folder